tests/data/all_ethN.biosdevname

```diff
@@ -0,0 +1,68 @@
+BIOS device: eth0
```

Comment:
Commenting here as github doesn't allow commenting commit messages: there's an unfinished sentence in the commit message: `Brings xcp. coverage from`